service/src/main/kotlin/fi/espoo/evaka/attachment/AttachmentsController.kt

```diff
@@ -142,6 +142,8 @@ class AttachmentsController(
         user.requireOneOfRoles(UserRole.END_USER)
         if (!db.read { it.isOwnAttachment(id, user) }) throw Forbidden("Permission denied")
 
+        s3Client.delete(filesBucket, "$id")
```

Comment:
Should you move this inside the transaction to reduce the chance of having state inconsistencies? In practice,
```
db.transaction {
    it.deleteAttachment(id)
    s3Client.delete(filesBucket, "$id")
}